components/script/dom/documentfragment.rs

```diff
@@ -77,8 +77,9 @@ impl DocumentFragmentMethods<crate::DomTypeHolder> for DocumentFragment {
 
     // https://dom.spec.whatwg.org/#dom-parentnode-children
     fn Children(&self) -> DomRoot<HTMLCollection> {
+        let can_gc = CanGc::note();
         let window = self.owner_window();
-        HTMLCollection::children(&window, self.upcast())
+        HTMLCollection::children(&window, self.upcast(), can_gc)
```

Comment:
Can you call `CanGc::note()` call directly in the call to `HTMLCollection::children`, without storing it in an intermediate variable first?

That is the convention we use everywhere else, and it makes it more obvious that this function needs a `CanGc` argument in the future.